docs/admin/telemetry.md

```diff
@@ -59,10 +61,31 @@ Property | Description
 When the data is collected, we also store the timestamp the data was received and
 a **hash** of the sending IP address - we do not store the plain value.
 
-## Schedule
 
-The platform will send the telemetry data:
+### Schedule
+
+For the core tracking, the platform will send the telemetry data:
  - 30 seconds after the platform starts up (but only if the platform has already been initialised)
  - Once every 24 hours at a time randomly picked when the platform starts
 
 The data is sent via an HTTP Post to `https://ping.flowforge.com`.
+
+## Plausible
+
+### Configuring Telemetry
+
+Further details about how users are navigating around your instance of FlowForge can be enabled via [Plausible](https://plausible.io/). This can be enabled with:
+
+```
+telemetry:
+  enabled: true
+  plausible:
+    domain: <data-domain>
+```
```

Comment:
No mention of the `extension: local` but it is mentioned in the configuration.md